src/cluster.c

```diff
@@ -834,22 +834,19 @@ void clusterCommand(client *c) {
 
     if (c->argc == 2 && !strcasecmp(c->argv[1]->ptr, "help")) {
         clusterCommandHelp(c);
-    } else if (!strcasecmp(c->argv[1]->ptr, "nodes") && c->argc == 2) {
+    } else if (!strcasecmp(c->argv[1]->ptr, "nodes") && c->argc <= 3) {
```

Comment:
NOTE to self: are all these checks really needed as we have command arity and schema validaitons?